GPU/GLES/GLES_GPU.cpp

```diff
@@ -1012,6 +1012,12 @@ void GLES_GPU::Execute_FramebufType(u32 op, u32 diff) {
 void GLES_GPU::Execute_ViewportType(u32 op, u32 diff) {
 	gstate_c.framebufChanged = true;
 	gstate_c.textureChanged |= TEXCHANGE_PARAMSONLY;
+	switch (op >> 24) {
+	case GE_CMD_VIEWPORTZ1:
+	case GE_CMD_VIEWPORTZ2:
+		shaderManager_->DirtyUniform(DIRTY_DEPTHRANGE);
+		break;
```

Comment:
I'm nitpicking now, but I would vote Execute_FramebufZType or something, though probably viewport commands are cheap enough it doesn't matter...

-[Unknown]